plugins/uniques.js

```diff
@@ -411,50 +550,71 @@ window.plugin.uniques.setupContent = function() {
+		+ '<label><input type="checkbox" id="scouted" onclick="window.plugin.uniques.updateScouted($(this).prop(\'checked\'))"> Scout controlled/label>'
+		+ '<label><input type="checkbox" id="droned" onclick="window.plugin.uniques.updateDroned($(this).prop(\'checked\'))"> Drone visited</label>'
 		+ '</div>';
 	plugin.uniques.disabledMessage = '<div id="uniques-container" class="help" title="Your browser does not support localStorage">Plugin Uniques disabled</div>';
 }
-
+// ***************************************************************************************
 window.plugin.uniques.setupPortalsList = function() {
 
 	window.addHook('pluginUniquesUpdateUniques', function(data) {
 		var info = plugin.uniques.uniques[data.guid];
-		if(!info) info = { visited: false, captured: false };
+		if (!info) info = { visited: false, captured: false, scouted: false, droned: false	};
 
 		$('[data-list-uniques="'+data.guid+'"].visited').prop('checked', !!info.visited);
 		$('[data-list-uniques="'+data.guid+'"].captured').prop('checked', !!info.captured);
+		$('[data-list-uniques="'+data.guid+'"].scouted').prop('checked', !!info.scouted);
+		$('[data-list-uniques="'+data.guid+'"].droned').prop('checked', !!info.droned);
 	});
 
 	window.addHook('pluginUniquesRefreshAll', function() {
 		$('[data-list-uniques]').each(function(i, element) {
 			var guid = element.getAttribute("data-list-uniques");
 
 			var info = plugin.uniques.uniques[guid];
-			if(!info) info = { visited: false, captured: false };
+			if (!info) info = { visited: false, captured: false, scouted: false, droned: false };
 
 			var e = $(element);
-			if(e.hasClass('visited')) e.prop('checked', !!info.visited);
-			if(e.hasClass('captured')) e.prop('checked', !!info.captured);
+			if (e.hasClass('visited')) e.prop('checked', !!info.visited);
+			if (e.hasClass('captured')) e.prop('checked', !!info.captured);
+			if (e.hasClass('scouted')) e.prop('checked', !!info.scouted);
+			if (e.hasClass('droned')) e.prop('checked', !!info.droned);
 		});
 	});
 
 	function uniqueValue(guid) {
 		var info = plugin.uniques.uniques[guid];
-		if(!info) return 0;
+		if (!info) return 0;
+		if (info.visited === undefined) return 0;
+		if (!info.visited) return 0;
+		if (info.visited && info.captured) return 2;
+		if (info.visited) return 1;
+	}
+
+	function scoutedValue(guid) {
+		var info = plugin.uniques.uniques[guid];
+		if (!info) return 0;
+		if (info.scouted === undefined ) return 0;
+		if (info.scouted === true) return 1;
+	}
 
-		if(info.visited && info.captured) return 2;
-		if(info.visited) return 1;
+	function dronedValue(guid) {
+		var info = plugin.uniques.uniques[guid];
+		if (!info) return 0;
+		if (info.droned === undefined ) return 0;
+		if (info.droned === true) return 1;
 	}
 
-	window.plugin.portalslist.fields.push({
-		title: "Visit",
+	window.plugin.portalslist.fields.push(
+		{title: "V/C",
```

Comment:
https://eslint.org/docs/rules/brace-style (1tbs)

Reply:
reason: having the { in the same line as "title" allows to fold the section in the editor.
nevertheless moved the { back to previous line.